share/spack/gitlab/cloud_pipelines/.gitlab-ci.yml

```diff
@@ -919,3 +894,36 @@ e4s-cray-sles-build:
   needs:
     - artifacts: True
       job: e4s-cray-sles-generate
+
+# aws-pcluster-build-neoverse_v1:
+#   extends: [  ".linux_neoverse_v1", ".aws-pcluster-neoverse_v1", ".build" ]
+#   trigger:
+#     include:
+#       - artifact: jobs_scratch_dir/cloud-ci-pipeline.yml
+#         job: aws-pcluster-generate-neoverse_v1
+#     strategy: depend
+#   needs:
+#     - artifacts: True
+#       job: aws-pcluster-generate-neoverse_v1
```

Comment:
Better double check on the current status of the pcluster jobs, I don't think they're commented out anymore, but I'm not sure.

Reply:
At one point I commented out the entire yaml except things required for Windows,  this is cruft.